ProcessMaker/Models/UserToken.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+
+namespace ProcessMaker\Models;
+
+
+use Laravel\Passport\Token;
+
+class UserToken extends Token
+{
+
+    /**
+     *
+     * @OA\Schema(
+     *   schema="UserToken",
+     *   @OA\Property(property="id", type="string"),
+     *   @OA\Property(property="name", type="string"),
+     *   @OA\Property(property="revoked", type="boolean"),
+     *   @OA\Property(property="created_at", type="string", format="date-time"),
+     *   @OA\Property(property="updated_at", type="string", format="date-time"),
+     *   @OA\Property(property="expires_at", type="string", format="date-time"),
+     * )
+     */
+
```

Comment:
@AbdelilahLbardi  Missing a few properties. 

**The needed properties:**
`user_id`
`client_id`
`scopes`
`revoked`
`client` set as the type object with its properties list as well. 
``` 
"client": {
        "id": 4,
        "user_id": null,
        "name": "ProcessMaker Personal Access Client",
        "provider": null,
        "redirect": "http://localhost",
        "personal_access_client": true,
        "password_client": false,
        "revoked": false,
        "created_at": "2022-05-04 17:23:26",
        "updated_at": "2022-05-04 17:23:26"
      }
```



**Example of the returned data**

```
{
      "id": "5c7d01f278d96",
      "user_id": 2,
      "client_id": 4,
      "name": "API Token",
      "scopes": [],
      "revoked": false,
      "created_at": "2022-05-05 15:32:16",
      "updated_at": "2022-05-05 15:32:16",
      "expires_at": "2023-05-05 15:32:16",
      "client": {
        "id": 4,
        "user_id": null,
        "name": "ProcessMaker Personal Access Client",
        "provider": null,
        "redirect": "http://localhost",
        "personal_access_client": true,
        "password_client": false,
        "revoked": false,
        "created_at": "2022-05-04 17:23:26",
        "updated_at": "2022-05-04 17:23:26"
      }
    }

Reply:
@AbdelilahLbardi  Just noticed that a new Model was for UserToken. This information can be put into the User model `ProcessMaker\Models\User;` as this is the model that is being used for UserToken.

Reply:
✅  done!